.github/workflows/graalpython_ci.yml

```diff
@@ -0,0 +1,38 @@
+# https://github.com/actions/setup-java
+# https://docs.github.com/en/actions/language-and-framework-guides/github-actions-for-java
+
+name: graalpython_ci
+on:
+  pull_request:
+  push:
+jobs:
+  graalpython_ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '14'
+      - run: java -version
+      - run: echo $JAVA_HOME
+      - run: git clone https://github.com/graalvm/mx.git
+      - run: cd graalpython ; ../mx/mx build  # 7+ minutes
+      - run: mx/mx python --version  # Python 3.8.5 (Interpreted JVM Development Build)
+      - run: mx/mx python -c "print(42)"  # 42
+      - run: mx/mx python-unittests || true  # ImportError: No module named html
+      - run: mx/mx python-src-import || true  # error: the following arguments are required: --cpython, --pypy, --python-version
+      - run: mx/mx python-unittests || true  # ImportError: No module named html
+      - run: mx/mx python-import-for-graal || true  # path = args[0] --> IndexError: list index out of range
+      - run: mx/mx python-unittests || true  # ImportError: No module named html
+      - run: mx/mx graalpython --version || true  # mx: unknown command 'graalpython'
+      - run: graalpython --version || true  # graalpython: command not found
+      - run: mx/mx python-svm -v || true  # 1 build tasks failed
+      - run: mx/mx python-svm -all || true  # 1 build tasks failed
+      - run: graalpython --version || true  # graalpython: command not found
+      - run: cd graalpython ; ../mx/mx python-svm -v || true  # 1 build tasks failed
+      - run: cd graalpython ; ../mx/mx python-svm -all || true  # 1 build tasks failed
+      - run: graalpython --version || true  # graalpython: command not found
+      - run: mx/mx python-gate || true  # 22+ minutes --> java.lang.OutOfMemoryError: Java heap space
```

Comment:
Basically, all of the commands from line 19 through line 34 should not be done in the CI (they are not run in our CI either, they are developer commands). `mx python-gate` is the only one that should run, but probably needs to be split up (in our CI we split it into ~20 jobs, and still some of these jobs take ~1h). So all we could reasonably do here is run a subset of the tests. We will likely never get rid of the internal CI anyway, so all these github actions could be are smoke tests.

Reply:
Probably a sufficient set of commands would be
```
cd graalpython
mx python-gate --tags style,fullbuild,python-unittest,python-unittest-jython
```
All our other gates are going to use so much memory and CPU time that it's unlikely to be useful.

Reply:
Yes.  I found it fascinating that none of the _developer commands_ actually work.

Reply:
they do work for developers ;) Some of them failed because of missing arguments (`mx python-src-import`, `mx python-import-for-graal`), some of the commands simply don't exist (for example, there is no global `graalpython` command, where would it come from?), got wrong arguments (`mx python-svm` does not take either `-all` or `-v` arguments, not sure why you thought it should?), and for `mx python-unittests` I suspect it was run with the wrong CPython version - that particular command is a brute force run of all python 3 library tests and currently needs to run with Python 3.8.5